app/src/views/TermsAndConditions/CondtionsModal/index.tsx

```diff
@@ -0,0 +1,265 @@
+            src_db_lab: strings.srcDbLabDescriptionThirteen,
+            src_org_code: strings.srcOrgCodeDescriptionThirteen,
+            src_org_lab: strings.srcOrgLabDescriptionThirteen,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionThirteen,
+            id: 13,
+        },
+        {
+            src_db_code: strings.srcDbCodeDescriptionFourteen,
+            src_db_lab: strings.srcDbLabDescriptionFourteen,
+            src_org_code: strings.srcOrgCodeDescriptionFourteen,
+            src_org_lab: strings.srcOrgLabDescriptionFourteen,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionFourteen,
+            id: 14,
+        },
+        {
+            src_db_code: strings.srcDbCodeDescriptionFifteen,
+            src_db_lab: strings.srcDbLabDescriptionFifteen,
+            src_org_code: strings.srcOrgCodeDescriptionFifteen,
+            src_org_lab: strings.srcOrgLabDescriptionFifteen,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionFifteen,
+            id: 15,
+        },
+        {
+            src_db_code: strings.srcDbCodeDescriptionSixteen,
+            src_db_lab: strings.srcDbLabDescriptionSixteen,
+            src_org_code: strings.srcOrgCodeDescriptionSixteen,
+            src_org_lab: strings.srcOrgLabDescriptionSixteen,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionSixteen,
+            id: 16,
+        },
+        {
+            src_db_code: strings.srcDbCodeDescriptionSeventeen,
+            src_db_lab: strings.srcDbLabDescriptionSeventeen,
+            src_org_code: strings.srcOrgCodeDescriptionSeventeen,
+            src_org_lab: strings.srcOrgLabDescriptionSeventeen,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionSeventeen,
+            id: 17,
+        },
+        {
+            src_db_code: strings.srcDbCodeDescriptionEighteen,
+            src_db_lab: strings.srcDbLabDescriptionEighteen,
+            src_org_code: strings.srcOrgCodeDescriptionEighteen,
+            src_org_lab: strings.srcOrgLabDescriptionEighteen,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionEighteen,
+            id: 18,
+        },
+        {
+            src_db_code: strings.srcDbCodeDescriptionNineteen,
+            src_db_lab: strings.srcDbLabDescriptionNineteen,
+            src_org_code: strings.srcOrgCodeDescriptionNineteen,
+            src_org_lab: strings.srcOrgLabDescriptionNineteen,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionNineteen,
+            id: 19,
+        },
+        {
+            src_db_code: strings.srcDbCodeDescriptionTwenty,
+            src_db_lab: strings.srcDbLabDescriptionTwenty,
+            src_org_code: strings.srcOrgCodeDescriptionTwenty,
+            src_org_lab: strings.srcOrgLabDescriptionTwenty,
+            src_orgtype_code: strings.srcOrgtypeCodeDescriptionTwenty,
+            id: 20,
+        },
+    ];
+
+    const tableKeySelector = (option:TableData) => option.id;
```

Comment:
Let's move this outside of the component. This is not dependent on any of the values in the component.